src/index.ts

```diff
@@ -119,7 +130,8 @@ async function main() {
     .addOption(disableNotificationsOption)
     .addOption(slackWebhookOption)
     .action((options) => {
-      const { logLevel } = options;
+      const { logLevel, orderBookApi } = options;
+
```

Comment:
As the `orderBookApi` isn't being modified here, or checked for any sanity, is it required to be deconstructed here? It should just pass in with the `...options` below, no?

Reply:
👍

Reply:
https://github.com/cowprotocol/watch-tower/pull/115